packages/big-design/src/components/InlineAlert/InlineAlert.tsx

```diff
@@ -27,12 +28,27 @@ export const InlineAlert: React.FC<InlineAlertProps> = memo(({ className, style,
 
   const renderedHeader = useMemo(() => header && <StyledHeader>{header}</StyledHeader>, [header]);
 
+  const renderedActions = useMemo(
+    () =>
+      props.actions && (
+        <Box marginTop="xSmall">
+          {props.actions.map(({ text, ...actionProps }, index) => (
```

Comment:
We probably want to ensure `children` can't get rendered.
```suggestion
          {props.actions.map(({ text, children, ...actionProps }, index) => (
```

Same applies for the other components.

Reply:
thank you for the comment 🙇 

in the current implementation, it works correctly!

As you see `children` from props is overrides by the content inside `Button` and if we will pass `undefined` as `text` and pass `children` from action props it's will be anyway overridden by the `undefined`.

You don't mind, I prefer to leave it as is.